orttraining/orttraining/python/training/onnxblock/engine/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .module import Module
```

Comment:
Can we come up with a more marketable name than engine?

Reply:
+1

Reply:
any suggestions?